arkane/main.py

```diff
@@ -102,14 +102,9 @@ def parseCommandLineArguments(self):
         sensible, parses them, and returns them.
         """
 
-        parser = argparse.ArgumentParser(description=
-                                         """
-        Arkane is a Python toolkit for computing chemical reaction rates and other
-        properties used in detailed kinetics models using various methodologies
-        and theories.
-        """)
-        parser.add_argument('file', metavar='FILE', type=str, nargs=1,
-                            help='a file describing the job to execute')
+        parser = argparse.ArgumentParser(description="""Arkane is a Python toolkit for computing chemical reaction 
+        rates and other properties used in detailed kinetics models using various methodologies and theories.""")
```

Comment:
Sorry, my previous comment wasn't very clear. I was referring to the opening quotes `"""` only. I think the line breaks in the text are likely intentional and should be preserved, since they transfer directly to the output in terminal.